mastoposter/utils.py

```diff
@@ -264,7 +264,7 @@ def node_to_plaintext(el: PageElement) -> str:
                     " \u2022 %s" % li.replace("\n", "\n   ").strip()
                     for li in children
                 )
-                if el.name == "ol"
+                if el.name == "ul"
```

Comment:
I should probably stop using lambdas everywhere and `a if c else b` thing, it makes everything unreadable and error-prone, totally missed that